content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -2,13 +2,21 @@
 GS1 barcode nomenclature
 ========================
 
-`GS1 nomenclature <https://www.gs1us.org/>`_ consolidates multiple pieces of information in a single
-barcode. Each piece needs to follow a specific barcode pattern—which is a defined format of numbers,
-letters, special characters, and character length—to ensure proper interpretation of the barcode. By
-scanning the barcode on an unopened box, GS1 nomenclature can identify the product, lot number,
-number of units contained, and more.
+.. _barcode/operations/gs1:
+
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+`GS1 nomenclature <https://www.gs1us.org/>`_ consolidates various product and supply chain data into
+a single barcode. Odoo takes in `unique Global Trade Item Numbers
+<https://www.gs1.org/standards/get-barcodes>`_ (GTIN), purchased by businesses, to enable global
+shipping, sales, and eCommerce product listing.
+
+Configure GS1 nomenclature to scan barcodes of sealed boxes and identify essential product
+information, such as |GTIN|, lot number, quantity information, and more.
```

Comment:
@StraubCreative Do you think this makes it clear enough that GTINs must be unique and purchased by GS1?